kibble/models/generic_item.go

```diff
@@ -26,6 +26,7 @@ type GenericItem struct {
 	Title    string
 	Images   ImageSet
 	Seo      Seo
+	StatusID int
```

Comment:
What does this default to for items other than films and tv? nil?

What is the status/int map?

Reply:
It seems its defaults to 0.
kibble doesn't really care about these values too much in terms of logic, but 1 = unpublished and 2 = published. It'll be 0 for all other generic items